src/applications/personalization/profile-2/components/direct-deposit/DirectDeposit.jsx

```diff
@@ -0,0 +1,37 @@
+import React, { useEffect } from 'react';
```

Comment:
This file is not entirely new. It was moved from the `components` dir into the new `components/direct-deposit` dir...